src/sentry/incidents/endpoints/serializers.py

```diff
@@ -626,7 +617,7 @@ def _validate_critical_warning_triggers(self, threshold_type, critical, warning)
         if threshold_type == AlertRuleThresholdType.ABOVE:
             alert_op = operator.lt
             threshold_type = "above"
-        elif threshold_type == AlertRuleThresholdType.BELOW:
+        else:
```

Comment:
pyright was complaining about unbound variables here, so just switched this to else